doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -277,9 +277,11 @@ name "/C110/"
 
 \end_inset
 
- Scalability and transferability need to be ensured.
- Beagle must be able to determine the performance of software running on
- an average desktop pc as well as running on a server or a cluster of servers.
+ Beagle must be able to ensure the transferability of its test results across
```

Comment:
Please do consitently talk of “measurements” not “tests” (As we are not testing, but measuring)